libs/sdk-core/src/breez_services.rs

```diff
@@ -2153,7 +2145,7 @@ pub(crate) mod tests {
         });
         let ln_invoice = receiver
             .receive_payment(ReceivePaymentRequest {
-                amount_sats: 3000,
+                amount_msat: 3000000,
```

Comment:
This can be written as `3_000_000` to improve readability

Reply:
done